### PR TITLE
Dev improves logging and fixes action parameters

### LINKED ISSUE #52: [BUG]Action parameters may be broken
**Describe the bug**
The charm does not have good parameters for its actions. When trying to execute an action, there is an error in validation because the parameter have underscores in their names (e.g. relation_id) and that throws an error.

**To Reproduce**
1- Execute any action with a parameter that contains an underscore. E.g. `juju run tls-certificates-operator/0 get-outstanding-certificate-requests relation_id=0`

**Expected behavior**
The action should execute without any errors.

**Screenshots**
image

**Environment**
 - tls-certificates-interface library version 2.12
 - Juju version: 3.1.5-genericlinux-amd64
 - Cloud Environment: Microk8s cloud
 - Kubernetes version 1.28

### PR DESCRIPTION
**Pull request by saltiyazan**
# Description

fixes #52
Improves parameter validation logging.
Changes action parameter names.

# Checklist:

- [ ] My code follows the [style guidelines](/CONTRIBUTING.md) of this project
- [ ] I have performed a self-review of my own code
- [ ] I have made corresponding changes to the documentation
- [ ] I have added tests that validate the behaviour of the software
- [ ] I validated that new and existing unit tests pass locally with my changes
- [ ] Any dependent changes have been merged and published in downstream modules

### CODE REVIEW

File: docs/how-to/h-provision-certificates.md [hunk 1]
```diff
@@ -10,13 +10,13 @@ To do so, the TLS Certificates Operator provides two actions:
 The following sections explain how to run both of them.
```
Comment (gruyaume) [inline docs/how-to/h-provision-certificates.md:10]: AFAIK those docs aren't synced to charmhub right now so I'd suggest simply deleting the `docs` directory from here and modifying the ones on discourse. 

If we don't do that we will very soon have a sync issue where what's here does not match what's on discourse. 

FYI @dariofaccin

**Review by gruyaume — COMMENTED** [review]
Let's try to use Jira issues number in the branch name (ex. `telco-732` in this case) so that this PR shows up in the Jira issue

**Comment by saltiyazan** [comment]
> Let's try to use Jira issues number in the branch name (ex. `telco-732` in this case) so that this PR shows up in the Jira issue

Good point, I agree. Is the comment a general one to pay more attention or should reopen the PR with that branch name?

File: docs/how-to/h-provision-certificates.md (hunk 1 above)
Comment (saltiyazan) [reply docs/how-to/h-provision-certificates.md:10]: What do you think @dariofaccin ?

**Comment by gruyaume** [comment]
> > Let's try to use Jira issues number in the branch name (ex. `telco-732` in this case) so that this PR shows up in the Jira issue
> 
> Good point, I agree. Is the comment a general one to pay more attention or should I reopen the PR with that branch name?

@saltiyazan More of a general comment, no need to create a new PR and loose the current comments/discussions.

**Review by gruyaume — COMMENTED** [review]
In the PR description, please specify the Github issue that the PR fixes, this will make it so that the issue will be closed once this PR is merged

```
fixes #52
```

File: docs/how-to/h-provision-certificates.md (hunk 1 above)
Comment (dariofaccin) [reply docs/how-to/h-provision-certificates.md]: I don't see any issues with removing the `docs` folder from here, until the sync action will be available. It also won't be a problem to cherry pick the folder in future, when we'll implement the sync.

**Review by dariofaccin — APPROVED** [review]

**Comment by saltiyazan** [comment]
Should we merge this and then update the topic on discourse? @gruyaume

**Review by gatici — APPROVED** [review]

**Review by gruyaume — APPROVED** [review]

**Pull request merged**